Search/src/com/FileReader.java

```diff
@@ -14,30 +14,30 @@ public static ArrayList<Doc> readFiles() {
 
         for (int i = 0; i < filesName.length; i++) {
                 File file = new File("tests/" + filesName[i]);
```

Comment:
folder name containing documents should be used as a config file. imagine that your folder name will be changed, then you should look around your codes to find where did you use "tests/" to change it.